# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: modules/libpref/init/StaticPrefList.yaml
# Commit: 8d3727e19ade
# Full Hash: 8d3727e19ade1f2373c3087b709ccb7a42249b6c
# Author: Tim Huang <tihuang@mozilla.com>
# Date: 2022-07-19 21:35:33
# Regressor Bug: 1773701
# File Overlap Count: 1
# Description:
#   Bug 1773701 - Part 2: Implement the email tracking data collection feature. r=dimi
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D151522
# ==============================================================================

diff -r fab41f0c857f -r 8d3727e19ade modules/libpref/init/StaticPrefList.yaml
--- a/modules/libpref/init/StaticPrefList.yaml	Tue Jul 19 14:46:07 2022 +0000
+++ b/modules/libpref/init/StaticPrefList.yaml	Tue Jul 19 14:46:08 2022 +0000
@@ -11725,6 +11725,12 @@
   value: @IS_NIGHTLY_BUILD@
   mirror: always
 
+# Collecting 3rd party emailtracking telemetry.
+- name: privacy.trackingprotection.emailtracking.data_collection.enabled
+  type: bool
+  value: true
+  mirror: always
+
 # Whether Origin Telemetry should be enabled.
 # NOTE: if telemetry.origin_telemetry_test_mode.enabled is enabled, this pref
 #       won't have any effect.